RecoTauTag/RecoTau/plugins/DeepTauId.cc

```diff
@@ -2580,9 +2803,15 @@ class DeepTauId : public deep_tau::DeepTauBase {
   const unsigned version_;
   const int debug_level;
   const bool disable_dxy_pca_;
+  const bool disable_hcalFraction_workaround_;
+  const bool disable_CellIndex_workaround_;
   std::unique_ptr<tensorflow::Tensor> tauBlockTensor_;
   std::array<std::unique_ptr<tensorflow::Tensor>, 2> eGammaTensor_, muonTensor_, hadronsTensor_, convTensor_,
       zeroOutputTensor_;
+  const bool save_inputs_;
+  std::ofstream* json_file_;
+  mutable bool is_first_block_;
```

Comment:
```suggestion
  bool is_first_block_;
```
this was meant to go together with making `saveInputs` non-const

Reply:
ah sorry. Indeed you said it.